WTWSMS/history/characters/greek.txt

```diff
@@ -2972,6 +2972,19 @@
 	}
 }
 
+1310016 = {
+	name="Solomon"
```

Comment:
Could you add a PLRE info comment, so it's easier to find him and avoid duplicates?

Reply:
Sure